.github/workflows/release.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       uses: actions/checkout@v4
       with:
         submodules: recursive
-    - uses: dtolnay/rust-toolchain@1.75.0
+    - uses: dtolnay/rust-toolchain@1.81.0
```

Comment:
Why is that change related to the PR?

Reply:
`soroban-sdk` 22.0.0-rc.3.2 requires rustc 1.79.0 or newer, and `soroban-cli` (which we use in the CI) now requires rustc 1.81.0 or newer.